packages/editor/src/lib/editor/managers/HistoryManager.ts

```diff
@@ -68,13 +69,14 @@ export class HistoryManager<
 
 			const { data, ephemeral, squashing, preservesRedoStack } = result
 
-			this.ignoringUpdates((undos, redos) => {
-				handle.do(data)
-				return { undos, redos }
-			})
+			// this.ignoringUpdates((undos, redos) => {
+			handle.do(data)
+			// 	return { undos, redos }
+			// })
```

Comment:
@ds300 I needed to do some changes here to allow side effects to produce undos. I don't think this had any other effects, but called out for your attention